.github/workflows/sonarcloud.yml

```
@@ -48,13 +48,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: '20'
-
-      - name: Install frontend dependencies and run Vitest
-        run: |
-          cd frontend
-          npm install
-          npm run test -- --coverage
-
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)